### DISCUSSION TITLE
Storing files outside of the members folder.

### DISCUSSION THREAD

**Discussion by jer-fortenberry** [spatie/dropbox-api#101] [General]
We are looking to store files outside of the default members folder and direct all the adapter methods (write/read/copy/etc) to folders that exist in the root of the Dropbox team folder. DropBox allows this by using the [Dropbox-API-Path-Root header](https://developers.dropbox.com/dbx-team-files-guide). 

In the [Client.php class](https://github.com/spatie/dropbox-api/blob/main/src/Client.php) the method getHeaders() seems to define using the $teamMemberID passed in from the construct. 

Unless I am missing something, I feel like this is the most logical choice to add the Dropbox-API-Path-Root header to accomplish our needs. That said, curious if you guys have any suggestions on best implementation or if its something you'd be interested in adding into a release before I fork the project for something so simple that potentially could be useful to others. 

Thanks for this project, greatly appreciated.

**Comment by freekmurze** [comment]
Free to create a PR that adds this functionality. Make sure to update the readme and tests. 👍

**Comment by p-w** [comment]
Check #105, this should be fixed :-)
